src/golang/build/golang.sh

```diff
@@ -36,7 +36,7 @@ export_path "/usr/local/go/${GOLANG_VERSION}/bin:\$GOPATH/bin"
 
 mkdir -p "$GOPATH/src" "$GOPATH/bin" "$GOPATH/pkg"
 
-chown -R ubuntu $GOPATH
+chown -R ${USER_ID} $GOPATH
```

Comment:
Better than `USER_NAME`?

Reply:
why? I think it doesn't matter as both need to match anyways